src/main/CommandLine.cpp

```diff
@@ -797,6 +801,53 @@ runRebuildLedgerFromBuckets(CommandLineArgs const& args)
     });
 }
 
+int
+runSimulate(CommandLineArgs const& args)
+{
+    CommandLine::ConfigOption configOption;
+    uint32_t desiredOps = 0;
+    uint32_t startLedger = 0;
+    uint32_t stopLedger = 0;
+    std::string historyArchiveGet;
+    return runWithHelp(
+        args,
+        {configurationParser(configOption),
+         clara::Opt{desiredOps, "DESIRED-OPERATIONS"}["--desired-operations"](
```

Comment:
why not just calling this something a little easier to understand like `--ops-per-ledger` (and `OPS-PER-LEDGER`)